src/webpack/webpack.common.config.js

```diff
@@ -19,7 +19,7 @@ export default {
             alwaysWriteToDisk: true,
         }),
         new InterpolateHtmlPlugin(HtmlWebpackPlugin, {
-            PUBLIC_URL: publicUrl,
+            PUBLIC_URL: (process.env.BASE_PATH ?? '/') + publicUrl.substr(1),
```

Comment:
Hvorfor måtte vi endre denne?

Reply:
For manifest.json og favicon i html-dokumentet. Akkurat nå hentes de fra `https://www.nav.no/public/{manifest.json,favicon}`